src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+* Added support for exporting metrics via
+  [user_events](https://docs.kernel.org/trace/user_events.html) on Linux when
+  OTLP protobuf encoding is enabled via the
+  `PrivatePreviewEnableOtlpProtobufEncoding=true` connection string switch.
+  ([#2113](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2113))
```

Comment:
With this, `PrivatePreviewEnableOtlpProtobufEncoding=true` is now supported in both Widows and Linux. Windows uses ETW as transport, while Linux uses user-events as transport.